src/stories/shared/select/MultiSelect.stories.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react'
```

Comment:
```suggestion
import { useState } from 'react'
```
React를 import에서 제거하는 게 어떨까용

Reply:
확인했습니다 !